domain/domain.go

```diff
@@ -624,7 +624,7 @@ func (do *Domain) refreshMDLCheckTableInfo() {
 	}
 	defer do.sysSessionPool.Put(se)
 	exec := se.(sqlexec.RestrictedSQLExecutor)
-	rows, _, err := exec.ExecRestrictedSQL(kv.WithInternalSourceType(context.Background(), kv.InternalTxnTelemetry), nil, "select job_id, version, table_ids from mysql.tidb_mdl_info")
+	rows, _, err := exec.ExecRestrictedSQL(kv.WithInternalSourceType(context.Background(), kv.InternalTxnTelemetry), nil, fmt.Sprintf("select job_id, version, table_ids from mysql.tidb_mdl_info where version <= %d", do.InfoSchema().SchemaMetaVersion()))
```

Comment:
why here add <= condition?  As I understand do.InfoSchema().SchemaMetaVersion() is max ver, right? then there is not biger ver in ddl jobs.